ui-new/html/deposit-network-down.html

```diff
@@ -0,0 +1,34 @@
+<section class="viewport deposit_network_down_state system-color" data-tr-section="deposit-network-down">
+  <section class="mainv">
+    <section class="header">
+      <img src="images/network-down.svg"/>
+      <h3 class="js-i18n">Network down</h3>
+    </section>
+    <section class="info">
+      <p class="d2 js-i18n">Sorry, we are down for the moment.</p>
```

Comment:
Just revised the screens and this text looks a lot more to what was in a screen called `trouble_state`, network down has slightly different text